frontend/components/common/BaseValueInput.vue

```diff
@@ -3,14 +3,17 @@
         <Tooltip :visible="!!errorMessage" placement="topLeft" :title="errorMessage">
             <Input
                 v-model="inputText"
-                class="Input"
-                :class="{ Error: !!errorMessage }"
+                class="Input pr-8"
+                :class="{ Error: !!errorMessage, 'pr-10': auctionType === 'surplus' }"
```

Comment:
Why do you need to add that here?

Reply:
See https://github.com/sidestream-tech/unified-auctions-ui/pull/334#discussion_r920869996 for explanation